### PR TITLE
Reporter: support all lazy finds; cleanup; better chunk loading

### PR DESCRIPTION
**Pull request by Nuckyz**

### CODE REVIEW

File: scripts/generateReport.ts [hunk 1]
```diff
@@ -116,13 +123,8 @@ async function printReport() {
             body: JSON.stringify({
                 description: "Here's the latest Vencord Report!",
                 username: "Vencord Reporter" + (CANARY ? " (Canary)" : ""),
-                avatar_url: "https://cdn.discordapp.com/icons/1015060230222131221/f0204a918c6c9c9a43195997e97d8adf.webp",
+                avatar_url: "https://cdn.discordapp.com/icons/1015060230222131221/6101cff21e241cebb60c4a01563d0c01.webp",
```
Comment (Vendicated) [inline scripts/generateReport.ts:126]:
```suggestion
                avatar_url: "https://cdn.discordapp.com/icons/1015060230222131221/6101cff21e241cebb60c4a01563d0c01.webp?size=512",
```

File: src/utils/lazyReact.tsx [hunk 2]
```diff
@@ -16,8 +16,12 @@ const NoopComponent = () => null;
  */
 export function LazyComponent<T extends object = any>(factory: () => React.ComponentType<T>, attempts = 5) {
     const get = makeLazy(factory, attempts);
-    return (props: T) => {
+    const component = (props: T) => {
```
Comment (Vendicated) [inline src/utils/lazyReact.tsx:19]: this gives it a proper displayName for react devtools
```suggestion
    const LazyComponent = (props: T) => {
```

File: src/webpack/webpack.ts [hunk 3]
```diff
@@ -91,7 +91,7 @@ if (IS_DEV && IS_DISCORD_DESKTOP) {
 }
 
 function handleModuleNotFound(method: string, ...filter: unknown[]) {
-    const err = new Error(`webpack.${method} found no module`);
+    const err = new Error(`webpack.${method} found no module.`);
```
Comment (Vendicated) [inline src/webpack/webpack.ts:94]: NO

Comment (Vendicated) [reply src/webpack/webpack.ts:94]: no periods allowed

Comment (Nuckyz) [reply src/webpack/webpack.ts:94]: mean

File: src/utils/lazyReact.tsx [hunk 4]
```diff
@@ -16,8 +16,12 @@ const NoopComponent = () => null;
...
         const Component = get() ?? NoopComponent;
         return <Component {...props} />;
     };
+
+    component.$$get = get;
```
Comment (Vendicated) [inline src/utils/lazyReact.tsx:24]:
```suggestion
    LazyComponent.$$get = get;
```

File: src/utils/lazyReact.tsx [hunk 5]
```diff
@@ -16,8 +16,12 @@ const NoopComponent = () => null;
...
+
+    return component;
```
Comment (Vendicated) [inline src/utils/lazyReact.tsx:26]:
```suggestion
    return LazyComponent;
```

File: src/webpack/webpack.ts (hunk 3 above)
Comment (Vendicated) [reply src/webpack/webpack.ts:94]:
```suggestion
    const err = new Error(`webpack.${method} found no module`);
```

**Pull request merged**